client/blocks/comments/helper.jsx

```diff
@@ -11,11 +11,7 @@ export function shouldShowComments( post ) {
 		return true;
 	}
 
-	if (
-		! post.is_jetpack &&
-		post.discussion &&
-		( post.discussion.comments_open || post.discussion.comment_count > 0 )
-	) {
+	if ( post.discussion && ( post.discussion.comments_open || post.discussion.comment_count > 0 ) ) {
```

Comment:
b4 merge lets do an isAutomattician check

Reply:
i thought we were not supposed to do those checks in the UI?

Reply:
why not? we already do that for a8c stream...

Reply:
hrm, i guess we have team checks elsewhere now..

Reply:
Doesn't the endpoint enforce that? (Jetpack comments for Automatticians only for the moment?)

Reply:
@bluefuton it does, and I think that's enough for now...

this may cause some confusion for HEs trying to diagnose issues with Atomic sites.